src/components/Menu/components/SettingsModal/SettingsModal.js

```diff
@@ -0,0 +1,36 @@
+import { Modal, StyleSheet, Text, Pressable, View, Image, TouchableHighlight } from "react-native";
+import styles from "./styles";
+
+const removeIcon = require("../../../../img/icons/remove_icon.png");
+
+const SettingsModal = ({ modalVisible, setModalVisible }) => {
```

Comment:
no need to pass `setModalVisible` to this component as it will only need to close it, and will never need to open